binding/react-native/ios/Porcupine.swift

```diff
@@ -86,4 +120,18 @@ class PvPorcupine: NSObject {
             reject("PvPorcupine:process", "Invalid Porcupine handle provided to native module.", nil)
         }
     }
+
+    private func extractResource(_ filePath: String) throws -> String {
```

Comment:
extractResource isn't really an accurate name anymore. Probably should call it getResourcePath or something